packages/react-ui-components/src/Tabs/tabs.tsx

```diff
@@ -106,8 +107,8 @@ export default class Tabs extends PureComponent<TabsProps> {
 
         return (
             <div className={theme!.tabs__content}>
-                {React.Children.map(children, (panel, index) => {
-                    const isActive = activeTab === index;
+                {children.map((panel, index) => {
```

Comment:
What's the difference between `React.Children.map` and `children.map` I keep on forgetting?

Reply:
I forgot again, but I had an issue with React.Children, I think it couldn't access something from the outside scope.